pages/common/diff-pdf.md

```diff
@@ -0,0 +1,16 @@
+# diff-pdf
+
+> Tool for visually comparing two PDFs.
```

Comment:
```suggestion
> Tool for comparing two PDFs.
```

Considering that the base command doesn't do this, I think its best to keep it removed.

Reply:
Also quoted from the creator: "diff-pdf is a tool for visually comparing two PDFs.", but I agree your suggestion is better.